ui/src/components/Search.js

```diff
@@ -35,6 +36,43 @@ export default props => (
                         />
                       </Col>
                       <Col className="search-results-wrapper" p={30} flex={1}>
+                        <Row
+                          bg="white"
+                          css={`
+                            height: 185px;
+                            padding: 16px;
+                          `}
+                        >
+                          <PrimarySiteChart sqon={sqon} />
+                          {/* <Col alignItems="center">
```

Comment:
gonna delete this?

Reply:
i'll make another PR with it to remember